spec/acceptance/acceptance_1_readme_spec.rb

```diff
@@ -66,7 +66,7 @@ class { 'java': }
       }
 
       tomcat::install { '/opt/tomcat7':
-        source_url => '#{TOMCAT7_RECENT_SOURCE}',
```

Comment:
It this used anywhere else? If yes should it be changed there too? If no, you can delete the definition too.